src/js/components/Notification/Notification.js

```diff
@@ -172,7 +172,15 @@ const Notification = ({
   if (message || actions)
     message =
       typeof message === 'string' ? (
-        <Message {...theme.notification.message} color={messageColor}>
+        <Message
+          {...theme.notification.message}
+          color={messageColor}
+          fill={
+            Message === Paragraph
+              ? theme.notification.message?.fill || false
+              : undefined
+          }
```

Comment:
I'm trying to think of a cleaner way to do this. It seems pretty awkward to first set Message to Paragraph or Text and then compare against Paragraph here. I think it would at least be better to set up props at the same time you define whether it's Paragraph or Text. It would probably be cleaner to actually pull this up into a separate Message component.

Reply:
I can take a stab at making this a bit cleaner